docker/hapi-compose.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.4'
+services:
+  db:
+    image: postgres
+    container_name: hapi-fhir-db
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: 'hapi'
+      POSTGRES_USER: 'admin'
+      POSTGRES_PASSWORD: 'admin'
+    volumes:
+      - hapi-fhir-db:/var/lib/postgresql/data
+    
+  hapi-server:
+    image: "hapiproject/hapi:latest"
+    container_name: hapi-server
+    environment:
+      - hapi.fhir.mdm_enabled=false
+      - hapi.fhir.enforce_referential_integrity_on_delete=false
+      - hapi.fhir.enforce_referential_integrity_on_write=false
```

Comment:
So as discussed offline, please retry with enabling referential integrity as my expectation is that the uploader should not violate it (and if it does, we should know). We disabled it for FHIR servers that are the sink for pipeline runs because we cannot guarantee referential integrity during pipeline runs.